src/components/Menu/Menu.tsx

```diff
@@ -10,13 +10,17 @@ import {
   UIComponentProps,
   ChildrenComponentProps,
   commonPropTypes,
+  getKindProp,
 } from '../../lib'
 import MenuItem from './MenuItem'
 import { menuBehavior } from '../../lib/accessibility'
 import { Accessibility } from '../../lib/accessibility/types'
 
 import { ComponentVariablesObject } from '../../themes/types'
-import { ReactProps, ShorthandValue } from '../../../types/utils'
+import { ReactProps, ShorthandCollection } from '../../../types/utils'
+import MenuDivider from './MenuDivider'
+
+export type MenuItemKindOptions = 'divider' | 'item'
```

Comment:
```suggestion
export type MenuItemKinds = 'divider' | 'item'
```

`MenuItemKinds` may be? Or even `MenuShorthandKinds` because they are related to shorthand items, not to `MenuItem`

Reply:
`MenuShorthandKinds` it is :)